src/createTippy.ts

```diff
@@ -317,7 +303,7 @@ export default function createTippy(
 
       if (
         instance.state.isVisible &&
-        includes(instance.props.trigger, 'click')
+        instance.props.trigger.indexOf('click') >= 0
```

Comment:
I kinda liked the `includes` helper more :P

Reply:
Popper 2 inlines `indexOf`, I think it's ok